dashboards/lodestar_networking.json

```
@@ -2003,28 +1830,32 @@
             "type": "prometheus",
             "uid": "${DS_PROMETHEUS}"
           },
+          "editorMode": "code",
           "exemplar": false,
-          "expr": "increase(lodestar_peers_requested_total_to_connect[$rate_interval])",
+          "expr": "60*rate(lodestar_peers_requested_total_to_connect[$rate_interval])",
           "interval": "",
           "legendFormat": "to connect",
+          "range": true,
           "refId": "A"
         },
         {
           "datasource": {
             "type": "prometheus",
             "uid": "${DS_PROMETHEUS}"
           },
+          "editorMode": "code",
           "exemplar": false,
-          "expr": "increase(lodestar_peers_requested_total_to_disconnect[$rate_interval])",
+          "expr": "60*rate(lodestar_peers_requested_total_to_disconnect[$rate_interval])",
           "hide": false,
           "interval": "",
           "legendFormat": "to disconnect {{reason}}",
+          "range": true,
           "refId": "B"
         }
       ],
       "thresholds": [],
       "timeRegions": [],
-      "title": "Requested peers to connect and disconnect",
+      "title": "Requested peers to connect and disconnect (rate / minute)",
```

consistency to other titles

```suggestion
      "title": "Requested peers to connect and disconnect (rate / min)",
```